docs/developers/development-guidelines/UINewSystem.md

```diff
@@ -0,0 +1,76 @@
+# UI Refactor (March 2025)
+
+
+
+## Introduction
+
+This document contains guidelines and tutorials for Stying and Templating the UI of the application.
+
+Depending on what the control will look like or what functionality is required will determine what method is used for styling. I considering this a type of flow chart for how complex we want to go.
+
+### Styling
+
+The most basic of visual changes when functionality stays the same. Styling is when the look of a control changes without changing any underlying markup and structure. Primarily used for color and font changes for various states but is used for layout changes such as margin, padding, alignment, spacing etc.
+
+For example, if we want to style an Avalonia `Seperator` control to match our system, we would create a new `SeperatorStyles.axaml` file inside of `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Separator`. Any created style files also need referencing in the `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/StylesIndex.axaml` file.
```

Comment:
Put on separate line to make it more apparent. Forgetting this happened to me too many times
```suggestion
For example, if we want to style an Avalonia `Seperator` control to match our system, we would create a new `SeperatorStyles.axaml` file inside of `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Separator`. 

Any created style files also need referencing in the `src/Themes/NexusMods.Themes.NexusFluentDark/Styles/StylesIndex.axaml` file.
```